tests/zfs-tests/tests/functional/cli_root/zfs_copies/zfs_copies_003_pos.ksh

```diff
@@ -50,12 +50,19 @@ function cleanup
 	if datasetexists $vol; then
 		log_must $ZFS destroy $vol
 	fi
+
+	if is_linux; then
+		echo 1 > /sys/module/zfs/parameters/zfs_vdev_parallel_open_enabled
+	fi
```

Comment:
Best to do it in a new PR too and remove those changes from this PR.  Then I'll be able to merge this.